loadtest/artillery/index.js

```diff
@@ -0,0 +1,125 @@
+import fs from 'fs'
+import childProcess from 'child_process'
+
+class artilleryTest {
+  errorMessage = []
+
+  constructor (testName) {
+    this.testName = testName.trim()
+    this.pathToFolder = `${__dirname}/tests/${this.testName}`
+    this.pathToProxy = `${__dirname}/tests/${this.testName}/proxy.js`
+    this.pathToTest = `${__dirname}/tests/${this.testName}/test.yml`
+    this.testResult = ''
+  }
+
+  /**
+   * init abd run test
+   *
+   * @returns {Promise<void>}
+   */
+  init = async () => {
+    this.isTestExists();
+    if (this.errorMessage.length > 0) {
+      console.log('Errors', this.errorMessage)
+    } else {
+      this.runProxy()
+      this.runTest()
+      this.log(this.testResult)
+    }
+  }
+  
+  /**
+   * Before run test, run proxy
+   *
+   * @returns {Promise<boolean>}
+   */
+  runProxy = async () => {
+    this.log('Run proxy test')
+    try {
+      const { runProxy } = require(this.pathToProxy);
+      await runProxy()
+    } catch (e) {
+      console.log(e);
+    }
+
+    return true
+  }
+
+  /**
+   * Run test
+   *
+   * @returns {Promise<void>}
+   */
+  runTest = async () => {
+    this.log('Run test')
+    this.testResult = childProcess.execSync(`artillery run ${ this.pathToTest}`).toString();
+  }
+
+  /**
+   * Checking test files
+   */
+  isTestExists = () => {
+    this.log('Start check file')
+
+      if (!fs.existsSync(this.pathToFolder)) {
```

Comment:
@StanislavShevchenko just a suggestion. Is it possible/allowed to create this folder if it not exists? Or it should be created manually?

Reply:
The rule for creating a test in the current implementation:
1 Folder name as in test
2 In this folder, a file named test.yml
3 In this folder, a file named proxy.js
If something is not there, then you will be informed